src/python/visualize_matching_result.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import cv2
+import argparse
+from pathlib import Path
+import protos_io
+
+
+def createHiddenMatchImage(width, height):
+    image = np.zeros((height, width, 3), dtype=np.uint8)
+    text_left_corner = (int(width / 3), int(height / 2))
+    image = cv2.putText(
+        image,
+        "No match found",
+        text_left_corner,
+        cv2.FONT_HERSHEY_SIMPLEX,
+        1,
+        (255, 255, 255),
+        2,  # thickness
+    )
+    return image
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--matching_result",
+        required=True,
+        type=Path,
+        help="Path to the matching result .MatchingResult.pb file",
+    )
```

Comment:
As of now this is not really intuitive, granted I know near to nothing about how the system works under the hood. But in the spirit of keeping it easy to use, I'd like either more explanation here or to be able to not need any internals-specific settings here. Not sure if this is possible though, so please treat it as just a suggestion.

Reply:
Yeah, I can see that.
This script, however, will be automatically called from another one. So hopefully, in the end, the user will not need to know the details. 
But I will leave it as it is for now.